console/src/main/resources/static/console-fe/src/pages/ConfigurationManagement/ConfigurationManagement/ConfigurationManagement.js

```diff
@@ -1384,7 +1400,7 @@ class ConfigurationManagement extends React.Component {
                 </div>
                 <Pagination
                   style={{ float: 'right' }}
-                  pageSizeList={ GLOBAL_PAGE_SIZE_LIST }
+                  pageSizeList={GLOBAL_PAGE_SIZE_LIST}
```

Comment:
please do not change the indent if you do not change this line

Reply:
Thank you for reviewing, I have committed the modification again